test/dogma/formatter/flycheck_test.exs

```diff
@@ -5,70 +5,37 @@ defmodule Dogma.Formatter.FlycheckTest do
   alias Dogma.Script
   alias Dogma.Error
 
-  with ".start" do
-    with "no files to list" do
-      should "print nothing" do
-        assert Flycheck.start([]) == ""
-      end
-    end
-
-    with "some files to lint" do
-      should "print nothing" do
-        script = [%Script{}, %Script{}, %Script{}]
-        assert Flycheck.start(script) == ""
-      end
-    end
-  end
-
-  with ".script" do
-    with "no errors" do
-      should "print nothing" do
-        script = %Script{ errors: [] }
-        assert Flycheck.script(script) == ""
-      end
-    end
-
-    with "some errors" do
-      should "print nothing" do
-        script = %Script{ errors: [%Error{}] }
-        assert Flycheck.script(script) == ""
-      end
+  with "no errors" do
```

Comment:
We should test the GenEvent functionality as well as the functions that format the strings.